README.md

```diff
@@ -99,7 +100,7 @@ our [guide](https://learning-at-home.readthedocs.io/en/latest/user/contributing.
 
 ## Citation
 
-If you found hivemind or its underlying algorithms useful for your research, please cite the relevant papers:
+If you found hivemind or its underlying algorithms useful for your research, please cite the following source:
```

Comment:
The specific source for the library is given below; saying "papers", giving one source and starting the paper discussion with "Also" is a bit misleading

Reply:
agreed